### PR TITLE
add hardwareId to tuf

### PR DESCRIPTION
**Pull request by simao**
do not start async listeners if disabled

### CODE REVIEW

File: core/src/main/scala/org/genivi/sota/core/Boot.scala [hunk 1]
```diff
@@ -214,28 +235,34 @@ object Boot extends BootApp
...
   val binding =
-    routes().flatMap { r =>
-      val sealedRoutes = sotaLog(Route.seal(r))
+    routes().flatMap { case (routes, updateNotifier) =>
+      val fullRoutes = routes ~ healthResource.route
+      val sealedRoutes = sotaLog(Route.seal(fullRoutes))
+
+      if (startAsyncListeners) {
+        if (tufClient.isDefined)
+          startTreehubCommitListener(db, updateNotifier, tufClient.get)
```
Comment (Danten) [inline core/src/main/scala/org/genivi/sota/core/Boot.scala]:
```
tufClient.foreach {
  startTreehubCommitListener(db, updateNotifier, _)
}

File: project/SotaBuild.scala [hunk 2]
```diff
@@ -236,9 +236,9 @@ object Dependencies {
 
   val JsonWebSecurityVersion = "0.4.5"
 
-  val libTufV = "0.0.1-93-gcfeb997"
+  val libTufV = "0.0.1-110-gf0219f8"
 
-  val libAtsV = "0.0.1-57-g00399d9"
+  val libAtsV = "0.0.1-64-g369e076"
```
Comment (Danten) [inline project/SotaBuild.scala]: Remember to update these

File: core/src/main/scala/org/genivi/sota/core/Boot.scala (hunk 1 above)
Comment (simao) [inline core/src/main/scala/org/genivi/sota/core/Boot.scala]: yeah, but I thought this would be more readable. I can change it in the next pr

Comment (Danten) [inline core/src/main/scala/org/genivi/sota/core/Boot.scala]: Fair enough

**Pull request merged**